services/data-info/src/data_info/routes/domain/common.clj

```diff
@@ -11,18 +11,16 @@
 (def DataIdPathParam (describe UUID "The data item's UUID"))
 
 (s/defschema Paths
-  {:paths (describe [NonBlankString] "A list of iRODS paths")})
+  {:paths (describe [(s/one NonBlankString "path") NonBlankString] "A list of iRODS paths")})
 
 (s/defschema OptionalPaths
-  (-> Paths
-    (->optional-param :paths)))
+  {(s/optional-key :paths) (describe [NonBlankString] "A list of iRODS paths")})
```

Comment:
This way OptionalPaths will still allow passing an empty list, as well as a missing key.